openwisp_controller/connection/api/urls.py

```diff
@@ -0,0 +1,26 @@
+from django.urls import path
+
+from . import views as api_views
+
+app_name = 'openwisp_controller'
+
+
+def get_api_urls(api_views):
+    """
+    returns:: all the API urls of the config app
+    """
+    return [
+        path(
+            'api/v1/controller/device/<uuid:id>/command/',
+            api_views.command_list_create_view,
+            name='api_device_command_list_create',
```

Comment:
I'd rename this as `device_command_list` for consistency (`connections_api` should be the namespace, I'll make sure @ManishShah120 adds the list functionality in https://github.com/openwisp/openwisp-controller/issues/464.